docs/source/system_reference_guide/s3-inventory.ipynb

```diff
@@ -0,0 +1,2025 @@
+{
```

Comment:
<p>stac_s3_map unused and I think we want separate lists for STAC collections in s3 and then collections in s3 but not in STAC</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>agreed ! </p>